tests/test_ModelSerialization.py

```diff
@@ -187,3 +197,42 @@ def test_foreign_key_exposed(self, Classroom):
         )
         assert len(classroom) == 1
         assert classroom[0]['school_id'] == self.school.id
+
+    def test_serialize_order_by_single_column(self, Student):
+        students = Student.serialize(
+            to_return=['name'],
+            order_by=Student.name
+        )
+        assert len(students) == 2
+        assert students[0]['name'] < students[1]['name']
+
+    def test_serialize_order_by_tuple(self, Student):
+        Student.serialize(
+            to_return=['name', 'email'],
+            order_by=(Student.name, Student.email)
+        )
+
+    def test_serialize_order_by_tuple_with_id(self, Student):
+        Student.serialize(
+            to_return=['name', 'email'],
+            order_by=(Student.name, Student.email, Student.id)
+        )
+
+    def test_serialize_with_limit_and_offset(self, Student):
+        student_a = Student.serialize(
+            to_return=['name'],
+            limit=1,
+            offset=0,
+            order_by=Student.name
+        )
+        assert len(student_a) == 1
+        assert student_a[0]['name'] in [self.student1.name, self.student2.name]
+        student_b = Student.serialize(
+            to_return=['name'],
+            limit=1,
+            offset=1,
+            order_by=Student.name
+        )
+        assert len(student_b) == 1
+        assert student_b[0]['name'] in [self.student1.name, self.student2.name]
+        assert student_a[0]['name'] != student_b[0]['name']
```

Comment:
<=?

Reply:
It's not testing the order they're returned in, it's testing that the offsetting returns a different model for this test